src/components/StoryCardComponent.vue

```diff
@@ -35,7 +35,8 @@ function navigateToStoryDetail() {
             {{ `${story.title}` }}
         </v-card-title>
         <v-card-text class="body-1">
-            {{ `${story.story.substring(0, 100)}... ` }}
+            <span v-if="showStory">{{ story.story }}</span>
```

Comment:
Can we take this out?  I think the showStory was there initially to do an expand/collapse of the card. I'm not sure it's going to do anything. Clicking on the story takes you to a different view.